src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -117,6 +118,7 @@
     "@types/lodash": "4.14.161",
     "@types/node": "12.12.62",
     "@types/simplebar": "5.1.1",
+    "@types/swagger-ui-dist": "3.30.0",
```

Comment:
I know that dep. "swagger-ui-dist" was added before this PR, but we should consider replacing it with ```swagger-ui```as the [documentation](https://github.com/swagger-api/swagger-ui#readme) says:

> We strongly suggest that you use swagger-ui instead of swagger-ui-dist if you're building a single-page application, since swagger-ui-dist is significantly larger.

Reply:
Thanks for the suggestion @alfonsomthd . Migrated to swagger-ui now